src/openassetio-core/src/hostApi/ManagerConveniences.cpp

```diff
@@ -0,0 +1,263 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2023 The Foundry Visionmongers Ltd
```

Comment:
[fix] 2024

Reply:
Good catch. Updated copyrights years in d03de4c9 6f8769d4 686f47a4